lang/en/local_edit_course_notification.php

```diff
@@ -27,6 +27,10 @@
 
 $string['notificationstatement'] = '';
 $string['notificationtype'] = 'Information';
+$string['notificationtype:error'] = 'Error';
+$string['notificationtype:information'] = 'Information';
```

Comment:
is it duplicate of $string['notificationtype'] ? or they are different?

Reply:
Actually $string['notificationtype'] is for default value.

Reply:
then you are doing it wrong. It should be 'info'